### PR TITLE
Games api

### PR DESCRIPTION
**Pull request by telrikk**

### CODE REVIEW

File: app/api/game.yaml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      parameters:
+        - name: offset
+          in: query
+          description: The offset at which the result set should begin.
+          required: false
+          type: integer
+          default: 0
+        - name: limit
+          in: query
+          description: Result will contain at most limit items
```
Comment (slifty) [inline app/api/game.yaml:18]: Total nit pick that DOESNT MATTER -- but you alternate on using punctuation in descriptions; really doesn't matter.

File: app/api/game.yaml [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+          description: Game was not found
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+definitions:
+  Game:
```
Comment (slifty) [inline app/api/game.yaml:87]: Worth listing who made a given game?

That could be a feature for a future iteration.

Comment (telrikk) [inline app/api/game.yaml:87]: Yeah, I do like that idea, but let's push it off to Asana for now and address it after 0.1 is released.

File: app/api/game.yaml (hunk 1 above)
Comment (telrikk) [inline app/api/game.yaml:18]: Haha, I agree. I think it makes sense for our documentation to be as polished as possible

**Pull request merged**